carservice/src/main/java/com/senlainc/warsaw/tyurin/controller/CraftsmanController.java

```diff
@@ -0,0 +1,74 @@
+package com.senlainc.warsaw.tyurin.controller;
+
+import com.senlainc.warsaw.tyurin.dto.CraftsmanDto;
+import com.senlainc.warsaw.tyurin.entity.Craftsman;
+import com.senlainc.warsaw.tyurin.exception.CriteriaNotFoundException;
+import com.senlainc.warsaw.tyurin.service.ICraftsmanService;
+import com.senlainc.warsaw.tyurin.util.builder.CraftsmanBuilder;
+import org.springframework.web.bind.annotation.*;
+
+import javax.validation.Valid;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping("craftsmen")
```

Comment:
There is some restful naming convention, check it out 
https://restfulapi.net/resource-naming/

Reply:
I read this article when I did this task :)
That's why I used "craftsmEn" as a collection resource, it's plural. But maybe I miss something else...

Reply:
Oh, that's good one :D
You are totally right here, my bad English the issue, not your endpoint :)